Gemfile

```diff
@@ -10,6 +10,8 @@ gem 'mongoid', '5.0.1'
 gem 'uglifier', '>= 1.3.0'
 gem 'jquery-rails', '~> 3.1.4'
 gem 'select2-rails', '~> 4.0.0'
+gem 'sidekiq', '~> 3.5.1'
```

Comment:
You want to be using https://github.com/alphagov/govuk_sidekiq here, which also removes the need for the initializer.